__tests__/app.test.js

```diff
@@ -222,3 +222,81 @@ describe('POST /api/reviews/:review_id/comments', () => {
   });
   });
 
+describe('PATCH /api/reviews/:review_id', () => {
+  test('should accept a positive number and update the votes property according to the number returning updated review', () => {
+    const votes = {inc_votes: 5}
+    const reviewFour = {
+      review_id: 4,
+      title: 'One Night Ultimate Werewolf',
+      category: 'hidden-roles',
+      designer: 'Akihisa Okui',
+      owner: 'happyamy2016',
+      review_body: 'We couldn\'t find the werewolf!',
+      review_img_url: 'https://images.pexels.com/photos/5350049/pexels-photo-5350049.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260',
+      created_at: '2021-01-18T10:01:41.251Z',
+      votes: 10
+    }
+    return request(app)
+    .patch('/api/reviews/4')
+    .send(votes)
+    .expect(200)
+    .then(({ body }) => {
+      expect(body).toEqual({review: [{
```

Comment:
why are we expecting your review to be an array? I would expect this to be an object, an array suggests that their might be multiples but this end point should only ever return one thing